packages/pwa-kit-extension-sdk/bin/list-overrides.js

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env node
```

Comment:
I'm wondering is we need to change the name of this bin. This will get added to a consuming projects `/node_modules/.bin` folder as `list-overrides` which could potentially have naming collisions since we aren't name spacing it. 

A possible path forward would be to call it "pwa-kit-extension-sdk". This would open it up to potentially house other commands like "adding an override for a file"...

Thoughts?

Reply:
I updated the script name to `pwa-kit-extension-sdk`, and kept the `pwa-kit-dev list-overrides` command using that script.